cmd/advertisement-operator/main.go

```diff
@@ -175,22 +184,23 @@ func main() {
 
 	// +kubebuilder:scaffold:builder
 
-	c := make(chan struct{})
 	var wg = &sync.WaitGroup{}
 	client, err := advertisementReconciler.InitCRDClient(localKubeconfig)
 	if err != nil {
 		os.Exit(1)
 	}
 	wg.Add(2)
-	go advertisementReconciler.CleanOldAdvertisements(c, wg)
+	go advertisementReconciler.CleanOldAdvertisements(componentStopper, wg)
 	go advertisementReconciler.WatchConfiguration(localKubeconfig, client, wg)
+	go newBroadcaster.WatchConfiguration(localKubeconfig, client, wg)
+	newBroadcaster.StartBroadcaster(componentStopper)
```

Comment:
Should be part of the WaitGroup in order to have a clean collection when the process is terminated.